packages/eslint-plugin/src/rules/no-mixed-enums.ts

```diff
@@ -170,18 +170,15 @@ export default createRule({
           .getSymbolAtLocation(tsNode)!
           .getDeclarations()!;
 
-        for (const declaration of declarations) {
-          for (const member of (declaration as ts.EnumDeclaration).members) {
-            return member.initializer
-              ? tsutils.isTypeFlagSet(
-                  typeChecker.getTypeAtLocation(member.initializer),
-                  ts.TypeFlags.StringLike,
-                )
-                ? AllowedType.String
-                : AllowedType.Number
-              : AllowedType.Number;
-          }
-        }
+        const [{ initializer }] = (declarations[0] as ts.EnumDeclaration)
```

Comment:
Love this because this exposes a bug (not sure if it's intentional)! Checking `declarations[0]` is almost always a bug.

Reply:
Is this truly a bug? Is it possible for a symbol to have multiple declarations, that would be different from each other?

Reply:
No; if a symbol returns multiple declarations, that means these sites are merged together and should be treated as one. That's my understanding; I haven't verified it.